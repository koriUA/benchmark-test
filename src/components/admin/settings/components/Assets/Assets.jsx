import React from 'react';

const Assets = () => <div>Here goes content for Assets</div>;

export default Assets;
