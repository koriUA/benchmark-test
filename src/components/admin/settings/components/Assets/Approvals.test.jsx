import React from 'react';
import { mount } from 'enzyme';

import Assets from './Assets';

describe('Assets', () => {
  test('should render the Assets', () => {
    const component = mount(<Assets />);
    expect(component).toMatchSnapshot();
  });
});
