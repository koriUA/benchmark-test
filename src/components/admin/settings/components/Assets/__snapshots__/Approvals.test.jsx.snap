// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Assets should render the Assets 1`] = `
<Assets>
  <div>
    Here goes content for Assets
  </div>
</Assets>
`;
